installer/src/main/java/com/teamscale/profiler/installer/steps/IStep.java

```diff
@@ -20,6 +20,11 @@ public interface IStep {
 	 */
 	void uninstall(IUninstallErrorReporter errorReporter);
 
+	/** Determines whether this step should not be run, e.g. because this step is not applicable to the current OS. */
+	default boolean shouldNotRun() {
```

Comment:
Nitpick: Should we maybe make this `shouldRun`? I always find this double negation a bit confusing to read.